Add basic user docs pages for each package (#156)
ros-controls/ros2_controllers: diff_drive_controller/doc/userdoc.rst

.. _diff_drive_controller_userdoc:

diff_drive_controller
---------------------

> Controller for differential drive wheel systems. Control is in the form of a velocity command, that is split then sent on the two wheels of a differential drive wheel base. Odometry is computed from the feedback from the hardware, and published.

Reviewer: ```suggestion
Controller for mobile systems with differential drive.
Input for control are velocity commands splitter for two wheels of the differential drive base.
Odometry is computed from the hardware's feedback and published.
```
